langserver/server.go

```diff
@@ -256,6 +256,25 @@ func (s *server) getPrometheusURL() string {
 	return s.PrometheusURL
 }
 
+// versionCompatibility checks if language server has access to new metadata APIs.
+func (s *server) versionCompatibility() (bool, error) {
```

Comment:
It would be good if the function name expressed that it's about the metadata API. Something like `supportsMetadataAPI`.

(nit)